Gemfile.lock

```diff
@@ -214,21 +214,19 @@ GEM
     method_source (0.9.2)
     mini_magick (4.11.0)
     mini_mime (1.1.0)
-    mini_portile2 (2.5.3)
     minitest (5.14.4)
     multi_json (1.15.0)
     multipart-post (2.0.0)
     nanaimo (0.3.0)
     nap (1.1.0)
     naturally (2.2.1)
     no_proxy_fix (0.1.2)
-    nokogiri (1.11.7)
-      mini_portile2 (~> 2.5.0)
+    nokogiri (1.11.7-x86_64-darwin)
```

Comment:
This (the use of the `x86_64-darwin` variant) is what makes the build on GitHubAction (which runs Danger on the code) fail to install nokogiri since the GHA image/machine we use is likely a Linux machine not a Mac/Darwin CPU.

We need to remove darwin from the list of platforms (or maybe instead add the platform used by GHA to the list of platforms) down below in that Gemfile.lock

Reply:
Ahhh looked right over that!